options.go

```diff
@@ -113,6 +117,14 @@ func InboundMessageHandler(handler func(c *Connection, message *iso8583.Message)
 	}
 }
 
+// ErrorHandler sets an ErrorHandler option
+func ErrorHandler(h func(err error)) Option {
```

Comment:
Will this contain SafeError values? We should mention that (after the PR is merged)

Reply:
If error is from the iso8583 package, then it most probably will be a `SafeError`. Also, I'll check places where we should use `SafeError` like this one (we don't know if sensitive information will be exposed, but it may):

```go
_, err = c.conn.Write([]byte(req.rawMessage))
if err != nil {
        c.handleError(fmt.Errorf("writing into connection: %w", err))
        break
}
```

Reply:
@adamdecaf I've updated the PR to use `SafeError` almost in all places.